project/cmake/modules/FindVAAPI.cmake

```diff
@@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
   unset(vaapi_version_str)
 endif()
 
+if(VAAPI_VERSION_STRING VERSION_LESS 0.38)
+message(FATAL_ERROR "VAAPI minimum version 0.38 not found")
+endif()
+
```

Comment:
I am not sure that actually happens - I need to make sure that we have at
least version 0.38 installed. If I read it correctly, without my line of
code - it will be succesful if any version is installed.

2016-08-01 19:57 GMT+02:00 Christian Fetzer <notifications@github.com>:

> In project/cmake/modules/FindVAAPI.cmake
> <https://github.com/xbmc/xbmc/pull/10170#discussion_r73022786>:
>
> > @@ -29,6 +29,10 @@ elseif(VAAPI_INCLUDE_DIR AND EXISTS "${VAAPI_INCLUDE_DIR}/va/va_version.h")
> >    unset(vaapi_version_str)
> >  endif()
> >
> > +if(VAAPI_VERSION_STRING VERSION_LESS 0.38)
> > +message(FATAL_ERROR "VAAPI minimum version 0.38 not found")
> > +endif()
> > +
>
> Nope :) No need to verify the version twice.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/xbmc/xbmc/pull/10170/files/584dc087d403b8f8af6f113d658d4ea4d99d5830..646ad3076596ec245ade7b52058d97d9dbbc0412#r73022786>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABCfHbKw5hylUc6sQhR5cVXkwsB8JCsFks5qbjN0gaJpZM4JUyTa>
> .
>



-- 
                   Key-ID:     0x1A995A9B
                   keyserver: pgp.mit.edu
==============================================================
Fingerprint: 4606 DA19 EC2E 9A0B 0157  C81B DA07 CF63 1A99 5A9B